### ISSUE TITLE
[CoE Starter Kit - Feature]: Consider Last Flow Run in Archival Flow Approval

### ISSUE THREAD

**Issue by nasilvae** [microsoft/coe-starter-kit#5214]
### Is your feature request related to a problem? Please describe.

![image](https://user-images.githubusercontent.com/95443860/230483075-151a8907-cb15-40ae-8ec6-dfe08b37464a.png)
Now that we have the data lake CoE live can we add the condition to consider flow last run in the past 6 months as this will allow the same level of automation of the app counter part. 

### Describe the solution you'd like

Archival flows to consider last flow run data.

**Comment by Jenefer-Monroe** [comment]
Hello, yes this is in our plans for a big rewrite of the Inactivity workflows. Its below some other priorities but we do want to do this soon. Will use this bug to have the request be explicity in the list.

**Comment by wfi-jonas** [comment]
yes please! I was surprised during rollout of COE that there was no tracking on flow runs for inactivity..

**Comment by LucX4** [comment]
Hi @Jenefer-Monroe, hope you are doing well!
Why are we not considering the turned-off Flows?

**Comment by maahi91** [comment]
Any update on this bug by when it gets resolved? If someone can provide an ETA that would be a great help.

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]:https://github.com/https://github.com/microsoft/coe-starter-kit/issues/10333

Thank you for your contributions
